<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EMMainWindow</class>
 <widget class="QWidget" name="EMMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emotion Mapper</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="mapGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>500</height>
        </size>
       </property>
       <property name="title">
        <string>Map</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="mapLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="agentInfoGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>500</height>
        </size>
       </property>
       <property name="title">
        <string>Agent info</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>N:</string>
        </property>
       </widget>
       <widget class="QLabel" name="nAgentLabel">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>20</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>90</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Emotions</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Goals</string>
        </property>
       </widget>
       <widget class="QListWidget" name="goalListWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>151</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="emotionListWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>151</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Coords:</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="coordsTextBrowser">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="beliefsListWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>366</y>
          <width>151</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>345</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Beliefs</string>
        </property>
       </widget>
       <widget class="QPushButton" name="emotionsPlotPushButton">
        <property name="geometry">
         <rect>
          <x>81</x>
          <y>83</y>
          <width>40</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
       <widget class="QPushButton" name="goalsPlotPushButton">
        <property name="geometry">
         <rect>
          <x>81</x>
          <y>193</y>
          <width>40</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
       <widget class="QPushButton" name="beliefsPlotPushButton">
        <property name="geometry">
         <rect>
          <x>81</x>
          <y>340</y>
          <width>40</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="emotionsPlotCheckBox">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>86</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="goalPlotCheckBox">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>196</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="beliefsPlotCheckBox">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>343</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="emotionsPlotAllPushButton">
        <property name="geometry">
         <rect>
          <x>53</x>
          <y>83</y>
          <width>29</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="controlsGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>581</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Controls</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Zoom</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="zoomHorizontalSlider">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="playPauseButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="stopButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QSpinBox" name="frameRateSpinBox">
          <property name="maximum">
           <number>400</number>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QLabel" name="frameRateLabel">
          <property name="text">
           <string>frame/sec</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="11">
         <widget class="QTextBrowser" name="currentTimeTextBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="12">
         <widget class="QSlider" name="mapSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="7">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="10">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>60</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="exportButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Export</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
